docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gliderlabs/logspout:v3.2.4 AS build-env
+ENV LOGSPOUT_SUMOLOGIC_VERSION 0.1.0-logspout-3.2.4-dev
+
+FROM alpine:3.7
+RUN apk add --update ca-certificates
+COPY --from=build-env /bin/logspout /bin/logspout
+ENTRYPOINT ["/bin/logspout"]
```

Comment:
Making this `CMD` rather than `ENTRYPOINT` makes it easier to override the command in `docker run` commands.

Reply:
Can't think we'd ever have a need for that. This is basically a single process image.

Reply:
If you want to poke around in the image you have to do:
```
docker run --rm -it --entrypoint sh praekeltfoundation/logspout-sumologic
```
...instead of just:
```
docker run --rm -it praekeltfoundation/logspout-sumologic sh
```

Not a big deal, tho.